examples/compiled/scatter_image.vg.json

```diff
@@ -34,8 +34,8 @@
       "from": {"data": "data_0"},
       "encode": {
         "update": {
-          "width": 50,
-          "height": 50,
+          "width": {"value": 50},
```

Comment:
Hmm, I now wonder if there a bug in the SVG preview / generator given that this Vega spec works properly fine in the vega editor.

Still, given we currently output xc/yc, this is still weird.

Reply:
With this PR and #5132, the editor is producing correct output:

![image](https://user-images.githubusercontent.com/111269/60386602-e95e3200-9a4b-11e9-8d32-d756985c5792.png)

The SVG generator is probably buggy.

Reply:
> The SVG generator is probably buggy.

Maybe it's just because the paths are not correct in the folder where we generate the images.

Reply:
Without #5132 images work but are behind the axes. Is that expected? 

![image](https://user-images.githubusercontent.com/589034/60744777-b5b56780-9f77-11e9-8d3f-1812c12767c5.png)

Reply:
Yes, expected, though maybe not desireable. (Need a separate issue to discuss this)